pkgs/development/python-modules/schwifty/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, iso3166
+, pycountry
+}:
+
+buildPythonPackage rec {
+  pname = "schwifty";
+  version = "2022.6.0";
+  format = "pyproject";
+
```

No obsolete Python releases are [supported](https://github.com/mdomke/schwifty/blob/master/setup.cfg#L34).

```suggestion

disabled = pythonOlder "3.7";
```